src/web3/ternoaConnect.tsx

```diff
@@ -38,6 +38,7 @@ if (!PROJECT_ID) {
 
 export default function TernoaConnect() {
     const { t } = useTranslation('global');
+    const navigate = useNavigate(); 
```

Comment:
Redirect, pas navigate !!!

It's recommended to use redirect in loaders and actions rather than useNavigate in your components when the redirect is in response to data.

Reply:
Il ne fonctionne pas, d'où le choix du navigate
![image](https://github.com/We-Are-Swissborg/WASB-Front/assets/101713837/d87e397e-1436-4b65-bc2e-dff0192cf4da)